build/tasks/RepoTasks.csproj

```diff
@@ -1,5 +1,5 @@
-<Project>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), version.props))\build\sdk.props" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepositoryRoot)files\KoreBuild\msbuild\KoreBuild.RepoTasks.Sdk\Sdk\Sdk.props" />
```

Comment:
Can build/sdk.* be removed?

Reply:
Not quite yet. I started doing that, but it required more changes to this repo than fit within the scope of "remove API check"